common/djangoapps/third_party_auth/lti.py

```diff
@@ -11,8 +11,8 @@
 from oauthlib.common import Request
 from oauthlib.oauth1.rfc5849.signature import (
     collect_parameters,
-    construct_base_string,
-    normalize_base_string_uri,
+    signature_base_string,
+    base_string_uri,
```

Comment:
These methods have been renamed in the latest version.

PR-651: https://github.com/oauthlib/oauthlib/pull/651